src/app/pages/material/material.component.ts

```diff
@@ -1,7 +1,6 @@
 import { Component, OnInit } from '@angular/core';
 import { state, style, transition, animate, trigger } from '@angular/animations';
 
-import { MaterialBookingFormComponent } from '../../components/material-booking-form/material-booking-form.component';
```

Comment:
Isn't it necessary to import it for using it in HTML file ?

Reply:
No